src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -59,23 +59,21 @@ static uint128 PassPhraseToKey (std::string const& passPhrase)
 
 static Blob getPublicKey (openssl::ec_key const& key)
 {
-	Blob result (33);
-	
-	key.get_public_key (&result[0]);
-	
-	return result;
+    Blob result (33);
```

Comment:
The code is being refactored as we speak.  The GenerateDeterministic-refactor branch moves the function out of RippleAddress, and I expect to ultimately remove it entirely.  This patch here is just a whitespace fix.